core/src/main/java/com/scalar/db/common/ActiveTransactionManagedDistributedTransactionManager.java

```diff
@@ -28,9 +31,8 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public abstract class ActiveTransactionManagedDistributedTransactionManager
-    extends TransactionDecorationDistributedTransactionManager
-    implements DistributedTransactionExpirationHandlerSettable {
+public class ActiveTransactionManagedDistributedTransactionManager
```

Comment:
Refactored this from using inheritance to using composition (decorator pattern).

Reply:
There is now `StateManagedDistributedTransactionManager` that can have Status.ACTIVE. So, this class prefix "Active" seems to related to the state and a bit confusing to me. I think this class provides the capability to resume and join a transaction, so how about renaming to something like `ResumableDistributedTransactionManager`?